pylint/extensions/check_ungrouped_isinstances.py

```diff
@@ -0,0 +1,46 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/master/COPYING
+
+"""Module to check ungrouped isinstances methods for pylint. """
+
+import astroid
+
+from pylint import checkers, utils
+from pylint.checkers.utils import check_messages
+from pylint.interfaces import IAstroidChecker
+
+MSGS = {
+    'E2685': ("isinstances of '%s' are not grouped. "
```

Comment:
Please rename it to consider-merging-isinstance and change the title and description of the message accordingly.